clang-tools-extra/clang-tidy/readability/AvoidNestedConditionalOperatorCheck.cpp

```diff
@@ -0,0 +1,52 @@
+//===--- AvoidNestedConditionalOperatorCheck.cpp - clang-tidy ----------===//
```

Comment:
Please add `-`

Reply:
fixed in 10602c2b4a662872d1aaeda1dbc58e5d6f613fda